src/components/MainComposer/components/MediaInputs/utils/mediaValidator/mediaValidator.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { mockFileImage, mockFileVideo } from '~mocks/mockMedias/mockFiles';
+
+import { MediaValidators } from './mediaValidators';
+
+const image = new MediaValidators(mockFileImage);
+const video = new MediaValidators(mockFileVideo);
+
+describe('check medias', () => {
```

Comment:
Isso aqui tem q ser o nome da classe, os testes não podem estar juntos assim no mesmo it, nesse caso vc nem tem q ter essas duas versoes, manda apenas um validator, nesse momento vc nao ta testando a diferença entre eles. Vc vai ter 2 validators quando vc for testar duração de uma media por exemplo, (que nao tem aqui vale dizer precisa por) tem q dar erro se tentar checar a duração de uma imagem e coisas assim